awscli/customizations/cloudformation/yamlhelper.py

```diff
@@ -53,12 +53,18 @@ def intrinsics_multi_constructor(loader, tag_prefix, node):
     return {cfntag: value}
 
 
+def str_representer(dumper, node):
```

Comment:
We don't need this, do we? customers having trouble with YAML integers can use JSON. I don't want to uglify YAML with quotes everywhere